modules/cudacodec/test/test_video.cpp

```diff
@@ -397,39 +397,35 @@ CUDA_TEST_P(VideoReadRaw, Reader)
 CUDA_TEST_P(CheckParams, Reader)
 {
     std::string inputFile = std::string(cvtest::TS::ptr()->get_data_path()) + "../highgui/video/big_buck_bunny.mp4";
-    {
-        cv::Ptr<cv::cudacodec::VideoReader> reader = cv::cudacodec::createVideoReader(inputFile);
-        double msActual = -1;
-        ASSERT_FALSE(reader->get(cv::VideoCaptureProperties::CAP_PROP_OPEN_TIMEOUT_MSEC, msActual));
-    }
```

Comment:
Should the tests which check for `CAP_PROP_OPEN_TIMEOUT_MSEC` etc. be added back as well?